crud/common/schema.lua

```diff
@@ -144,7 +144,7 @@ function schema.filter_obj_fields(obj, field_names)
 
     for _, field_name in ipairs(field_names) do
         result[field_name] = obj[field_name]
-        if result[field_name] == nil then
+        if type(result[field_name]) == nil then
```

Comment:
This condition is always false. Because type(...) returns a string. It could return `"nil"` (as string). But anyway as I mentioned in issue https://github.com/tarantool/crud/issues/140 this check won't protect you from cases when nullable field is not box.NULL it's just `nil`. It's possible to omit trailing nils in tuples or user could add some nullable fields at the end of a tuple during migration. My proposal is to remove this check at all.